packages/oidc-key-registry-server/src/config.ts

```diff
@@ -0,0 +1,6 @@
+import "dotenv/config";
+
+export default {
```

Comment:
In the governance app we used this to manage environment. Pretty cool lib: https://env.t3.gg/docs/core